### PR TITLE
Fix issues in the docs.

### PR DESCRIPTION
**Pull request by rimbi**
Fix issues in the docs:

- Updates the docs and help message with regards to sentinel table as it was moved from source databse to sqlite.
- Fixes minor grammatical issues in the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,22 +107,22 @@ $ pgcopydb help
...
     transform  Transform changes from the source database into SQL commands
     apply      Apply changes from the source database into the target database
 
   pgcopydb stream sentinel
-    create  Create the sentinel table on the source database
-    drop    Drop the sentinel table on the source database
-    get     Get the sentinel table values on the source database
-  + set     Maintain a sentinel table on the source database
+    create  Create the sentinel table
+    drop    Drop the sentinel table
```
Comment (dimitri) [inline README.md:117]: We don't have these two commands anymore, I believe this PR should also remove the entries from this list.

File: docs/update-help-messages.sh [hunk 2]
```diff
@@ -114,10 +114,10 @@ function parse_help_output() {
 
         # Parse subcommands that are followed by a section header. For example,
         # there are the subcommands under pgcopydb strem sentinel section:
-        #     create  Create the sentinel table on the source database
-        #     drop    Drop the sentinel table on the source database
-        #     get     Get the sentinel table values on the source database
-        #   + set     Maintain a sentinel table on the source database
+        #     create  Create the sentinel table
+        #     drop    Drop the sentinel table
```
Comment (dimitri) [inline docs/update-help-messages.sh:118]: I'm confused as of why we still have these.

**Review by dimitri — CHANGES_REQUESTED** [review]
LGTM, thanks!

There is only one catch with the `pgcopydb sentinel create|drop` CLIs that I believe do not exist anymore, right?

File: README.md (hunk 1 above)
Comment (rimbi) [reply README.md]: You are right, fixed. 👍

File: docs/update-help-messages.sh (hunk 2 above)
Comment (rimbi) [reply docs/update-help-messages.sh]: You are right, fixed. 👍

**Review by dimitri — APPROVED** [review]

**Pull request merged**